README.md

```diff
@@ -1,9 +1,37 @@
 # awesome-green-extensions
 A list of climate friendly extensions that can help you reduce the carbon footprint of your internet consumption. Work in progress. Needs links and descriptions and currently focused on Chrome.  Feel free to recommend more via Issue or ideally PR.
 
-## CO2 emissions trackers and green website raters
+# CO2 emissions trackers and green website raters
```

Comment:
This should be an H2 (##) because we are using H1(#) for the main title on line 1